src/gurt/misc.c

```diff
@@ -11,12 +11,36 @@
 
 #include <stdarg.h>
 #include <math.h>
+#include <malloc.h>
 #include <string.h>
 #include <gurt/common.h>
 
 /* state buffer for DAOS rand and srand calls, NOT thread safe */
 static struct drand48_data randBuffer = {0};
 
+d_alloc_track_cb_t d_alloc_track_cb;
+d_alloc_track_cb_t d_free_track_cb;
+static __thread void *track_arg;
+
+void
+d_set_alloc_track_cb(d_alloc_track_cb_t alloc_cb, d_alloc_track_cb_t free_cb)
+{
+	char	*env;
+
+	env = getenv(DAOS_MEM_TRACK_ENV);
```

Comment:
We should add an entry to src/cart/README.env (perhaps it should be D_ for consistency with gurt).

We should probably also add a section to docs/debugging.md

Reply:
There's a d_getenv_bool() function that could be used here isn't there?